src/target/riscv/riscv-013.c

```diff
@@ -498,6 +498,23 @@ static void increase_dmi_busy_delay(struct target *target)
 	dtmcontrol_scan(target, DTM_DTMCS_DMIRESET, NULL /* discard result */);
 }
 
+static void decrease_reset_delays_wait(struct target *target, size_t finished_scans)
```

Comment:
Please, help me find a more descriptive name for this function. How about `decrement_reset_delays_counter()`, `handle_reset_delays_wait_counter()`, `reset_learned_delays_if_needed()` or similar.

Reply:
Addressed.